src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -295,6 +295,10 @@ class CollapsibleCourseRun extends React.Component {
           disabled={courseInReview}
           required={courseRunSubmitting}
         />
+        <div>
+          <FieldLabel id="number" text="Add OFAC Restriction text to the FAQ section" className="mb-2" />
```

Comment:
Might also be worth setting helpText for the FieldLabel so that people that don't know what OFAC is can see a nice tooltip that might calm any confusion.